### DISCUSSION TITLE
@react-three/* not available on npm

### DISCUSSION THREAD

**Discussion by Leandro201927** [pmndrs/drei#2262] [Q&A]
Someone can explain me what happened to @react-three/* libraries? there is not available anymore on npm registry.
![image](https://github.com/user-attachments/assets/50121bbf-b160-4509-8512-9126f716ec31)

This is my package.json:
![image](https://github.com/user-attachments/assets/eb61dc6e-0c40-4022-9832-eecca0685abe)

Logs:
![image](https://github.com/user-attachments/assets/312e9a69-b568-48a0-a35a-2e562116265b)

**Comment by netgfx** [comment]
What do you mean? I can see this: https://www.npmjs.com/package/@react-three/fiber
and this https://www.npmjs.com/package/@react-three/drei/v/9.0.1 package just fine
Maybe you are behind some VPN or something?
